stl/inc/xutility

```diff
@@ -698,9 +698,12 @@ namespace ranges {
             template <class _Ty>
                 requires (_Choice<_Ty>._Strategy != _St::_None)
             _NODISCARD constexpr decltype(auto) operator()(_Ty&& _Val) const noexcept(_Choice<_Ty>._No_throw) {
-                if constexpr (_Choice<_Ty>._Strategy == _St::_Custom) {
+

```

Comment:
There are still some more newlines, please remove all extra newlines before the `_Strat ` variable

Reply:
Ok